configure

```diff
@@ -1480,14 +1480,37 @@ sub write_opendds_configh {
     $exclusive = 0;
   }
 
+  my $cxx11 = 0;
+  for my $f (@features) {
+    if ($f eq 'no_cxx11=0') {
+      $cxx11 = 1;
+    }
+  }
+
   my %config = (
     'OPENDDS_CONFIG_AUTO_STATIC_INCLUDES' => 0,
     'OPENDDS_CONFIG_BOOTTIME_TIMERS' => $opts{'boottime'} // 0,
+    'OPENDDS_CONFIG_BUILT_IN_TOPICS' => $opts{'built-in-topics'} // 1,
+    'OPENDDS_CONFIG_CONTENT_FILTERED_TOPIC' => $opts{'content-filtered-topic'} // ($opts{'safety-profile'} ? 0 : 1),
```

Comment:
If we have a `--no-content-subscription` configuration, is `OPENDDS_CONFIG_CONTENT_FILTERED_TOPIC` set to 0 (along with multi topic and query condition)?

Reply:
No.  There is a check that content subscription is enabled if any of content filtered, multi topic, or query condition are enabled.  https://github.com/jrw972/OpenDDS/blob/4e1251a65634bb9058e0fa3b1ede156264608e26/dds/OpenDDSConfigWrapper.h#L201

If we want to deal with this in the configure script, then I think it would be better to enable content subscription if any of the dependents are enabled.

Reply:
Everything defaults to enabled (except with safety profile), so I think the more common use case is the one I described above.  The logic in `dcps_optional_features.mpb` that this PR is removing/replacing says that if `content_subscription == 0` then the 3 sub-features get turned off.

So it may be good to change the configure script, or maybe change the wrapper to do the inverse of this block:

https://github.com/jrw972/OpenDDS/blob/4e1251a65634bb9058e0fa3b1ede156264608e26/dds/OpenDDSConfigWrapper.h#L195

If the user configures `OPENDDS_CONFIG_CONTENT_SUBSCRIPTION == 0` then it turns off the 3 sub-features.

Reply:
Maybe you have this already in the latest revision?

Reply:
See `MPC\config\dcps_optional_features.mpb` -- the umbrella feature is not a preprocessor macro, just a shortcut to get all 3 other macros set